### PR TITLE
Adding "Gist Created" Notification

### PR DESCRIPTION
**Pull request by StanleyGoldman**
Adding a simple "Gist Created" notification as requested in #196

### CODE REVIEW

File: src/GitHub.App/ViewModels/GistCreationViewModel.cs [hunk 1]
```diff
@@ -8,6 +8,7 @@
 using GitHub.Extensions;
 using GitHub.Models;
 using GitHub.Services;
+using Microsoft.VisualStudio.Shell;
```
Comment (shana) [inline src/GitHub.App/ViewModels/GistCreationViewModel.cs]: No need for this change :)

**Comment by shana** [comment]
Thank you for the PR! Apologies on the radio silence, we've been traveling and it's disrupted our flow a bit, but it's not forgotten 😄

File: src/GitHub.App/ViewModels/GistCreationViewModel.cs (hunk 1 above)
Comment (StanleyGoldman) [inline src/GitHub.App/ViewModels/GistCreationViewModel.cs]: How did that get there...
Removed.

**Comment by StanleyGoldman** [comment]
Fixed and rebased.

**Comment by grokys** [comment]
LGTM, but it's out of date with master again :( Do you want to enable us pushing to the branch so we can do the merging with master?

**Comment by StanleyGoldman** [comment]
I'll rebase it in a little bit

**Comment by StanleyGoldman** [comment]
Rebased

**Comment by grokys** [comment]
Thanks Stanley! 🎉 It may take us a while, but we do merge eventually ;)

**Pull request merged**